jobs/fde.md

```diff
@@ -0,0 +1,77 @@
+# Forward Deployed Engineer
+
+## About Us
+
+Teleport delivers trusted computing to modern infrastructure.
+
+Teleport is trusted by the world's most innovative companies. Teleport is the
+Infrastructure Identity Company, modernizing identity, access, and policy for
+infrastructure, improving engineering velocity and resiliency of critical
+infrastructure against human factors and/or compromise.
+
+Teleport is a fast-growing, well-funded Y-Combinator company. We value our
+craft, are strong supporters of work/life balance, and embrace a culture of
+humility, honesty, and transparency.
```

Comment:
This section feels a little bit repetitive. We start almost every sentence with "Teleport is". Any desire to remove some of the fluff here?